overdrive.py

```diff
@@ -127,11 +128,14 @@ class OverdriveAPI(object):
 
     EVENT_DELAY = datetime.timedelta(minutes=120)
 
-    # The ebook formats we care about.
-    FORMATS = "ebook-epub-open,ebook-epub-adobe,ebook-pdf-adobe,ebook-pdf-open"
+    # The formats we care about.
+    FORMATS = "ebook-epub-open,ebook-epub-adobe,ebook-pdf-adobe,ebook-pdf-open,audiobook-overdrive".split(",")
 
     # The formats that can be read by the default Library Simplified reader.
-    DEFAULT_READABLE_FORMATS = set(["ebook-epub-open", "ebook-epub-adobe"])
+    DEFAULT_READABLE_FORMATS = set(
+        ["ebook-epub-open", "ebook-epub-adobe", "ebook-pdf-open", 
```

Comment:
DRM-free PDF support was added to SimplyE a while ago; this updates the Overdrive client to notice those files in a library's Overdrive collection.